pom.xml

```diff
@@ -701,6 +702,13 @@
             <name>!skip.code-coverage</name>
         </property>
       </activation>
+      <properties>
+          <sonar.dynamicAnalysis>reuseReports</sonar.dynamicAnalysis>
+          <sonar.surefire.reportsPath>${project.build.directory}/surefire-reports</sonar.surefire.reportsPath>
+          <sonar.coverage.jacoco.xmlReportPaths>
+              ${project.build.directory}/site/jacoco/jacoco.xml
```

Comment:
Why don't we have `phoenix-queryserver-assembly` here like for the phoenix main repo?

Reply:
If I do it like in the phoenix component and aggregate the coverage in the assembly module then the phoenix-queryserver-orchestrator module is not included in the coverage however I tried to modified the configuration of the jacoco and sonar profiles.